src/apyfixedarray.h

```diff
@@ -141,9 +152,26 @@ class APyFixedArray : public APyBuffer<mp_limb_t> {
     APyFixedArray broadcast_to_python(const std::variant<nb::tuple, nb::int_> shape
     ) const;
 
+    //! Returns a copy where all axes of size 1 is removed.
     APyFixedArray
     squeeze(std::optional<std::variant<nb::int_, nb::tuple>> axis = std::nullopt) const;
 
+    //! Returns a copy where the specified axes is summated.
+    APyFixedArray
+    sum(std::optional<std::variant<nb::int_, nb::tuple>> axis = std::nullopt) const;
```

Comment:
The return type of `sum` and `nansum` can be either a scalar or an array, ([link to numpy docs](https://numpy.org/doc/stable/reference/generated/numpy.sum.html)). This can be achieved using `std::variant`, see `APyFixedArray::get_item` as an example.